speech/setup.py

```diff
@@ -51,6 +51,8 @@
 
 REQUIREMENTS = [
     'google-cloud-core >= 0.20.0',
+    'gapic-google-cloud-speech-v1beta1==0.11.1',
+    'grpc-google-cloud-speech-v1beta1==0.11.1',
```

Comment:
See the link I provided, you also want upper bounds.

The change from https://github.com/GoogleCloudPlatform/google-cloud-python/commit/8632578 not showing up here?